<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReverbSettings</class>
 <widget class="QDialog" name="ReverbSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>110</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #202020;
color: #FFFFFF;</string>
  </property>
  <widget class="QDial" name="mix">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>24</y>
     <width>90</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mix</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mixValue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>82</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Roomsize</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDial" name="roomsize">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>24</y>
     <width>90</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="roomsizeValue">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>82</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDial" name="width">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>24</y>
     <width>90</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="widthValue">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>82</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Damp</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDial" name="damp">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>24</y>
     <width>90</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="dampValue">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>82</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
